src/backend/distributed/sql/downgrades/citus--12.2-1--12.1-1.sql

```diff
@@ -51,3 +50,15 @@ DROP FUNCTION citus_internal.update_relation_colocation(oid, int);
 DROP FUNCTION citus_internal.start_replication_origin_tracking();
 DROP FUNCTION citus_internal.stop_replication_origin_tracking();
 DROP FUNCTION citus_internal.is_replication_origin_tracking_active();
+
+DROP VIEW pg_catalog.citus_shards;
+#include "../udfs/citus_shards/12.0-1.sql"
+
+DROP FUNCTION citus_internal.add_shard_metadata(regclass, bigint, "char", text, text, boolean);
+DROP FUNCTION pg_catalog.citus_internal_add_shard_metadata(regclass, bigint, "char", text, text, boolean);
+#include "../udfs/citus_internal_add_shard_metadata/10.2-1.sql"
```

Comment:
regarding https://github.com/citusdata/citus/pull/7193/files#r1484111841, is this correct?